### PR TITLE
Added Pangolin Connector

### PR DESCRIPTION
**Pull request by prodesert22**

### CODE REVIEW

File: contracts/avalanche/connectors/pangolin/main.sol [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+     * @dev Buy AVAX/ERC20_Token.
+     * @notice Buy a token using a Pangolin
+     * @param buyAddr The address of the token to buy.(For AVAX: 0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE)
+     * @param sellAddr The address of the token to sell.(For AVAX: 0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE)
+     * @param buyAmt The amount of tokens to buy.
+     * @param unitAmt The unit amount of sellAmt/buyAmt with slippage.
+     * @param getId ID to retrieve buyAmt.
+     * @param setId ID to store the amount of tokens sold.
+    */
+    function buy(
```
Comment (benber86) [inline contracts/avalanche/connectors/pangolin/main.sol:95]: Does DSA offer buys/sales ?

**Review by benber86 — CHANGES_REQUESTED** [review]
Nice, could you add tests?

File: contracts/avalanche/connectors/pangolin/main.sol (hunk 1 above)
Comment (prodesert22) [reply contracts/avalanche/connectors/pangolin/main.sol:95]: I believe it's for swap
https://docs.instadapp.io/connectors/mainnet/uniswap#buy

**Comment by prodesert22** [comment]
> Nice, could you add tests?

Ok, I'll add

**Review by benber86 — APPROVED** [review]
lgtm

**Pull request merged**